### ISSUE TITLE
Could you teach me how to use it

### ISSUE THREAD

**Issue by whyimsodiao** [Lukrecjaaa/nbtmcafix#1]
My archive is constantly crashing due to this issue, I am a complete newcomer and I won't be able to use this code. Could you tell me how to use it, which files need to be executed, and which parameters need to be modified? thank you so much！

**Comment by Lukrecjaaa** [comment]
Hii, sure, first of all you need to download the binary from here: https://github.com/Lukrecjaaa/nbtmcafix/releases/download/1.0/nbtmcafix.jar

After that, make a backup of your Minecraft world just in case and open a terminal (I'm assuming you have Java installed). Go to the directory where the binary is located and execute the following command:
```
java -jar nbtmcafix.jar <location of your Minecraft server directory>/world
```

You can also set `-e` and `-p` flags, they specify the maximum allowed depth of NBT objects, but it should work just fine with default settings (max depth of 512).

**Comment by whyimsodiao** [comment]
Can it only be used on the server? I filled in my archive directory there, and he replied to me Invalid directories. it's  a single modpackage.

**Comment by Lukrecjaaa** [comment]
I'm not sure if I understand what do you mean by "archive directory", could you explain it in a bit more detail?

If you meant that your single-player world is corrupted, you might try something like:
```
java -jar nbtmcafix.jar "<location of your .minecraft directory>\saves\<your world name>"
```

Just make a backup of the world first. I'm not sure if it's going to work, since I wrote the program for my use case, which was to fix a server world, but there's a high probability it's capable of fixing single-player worlds as well.

**Comment by whyimsodiao** [comment]
It seems not working. But I'm really thank you. 
![image](https://github.com/Lukrecjaaa/nbtmcafix/assets/52467744/6c9794ad-4775-4800-95c9-4e4b340fca2d)

**Comment by Lukrecjaaa** [comment]
Ah, that's unfortunate.. Maybe that's because I wrote this program to fix 1.19.x worlds and you're trying to fix a world from an older version.

**Comment by whyimsodiao** [comment]
yeah, I use it on 1.12.2. Thanks for you help again!

**Comment by Xplodin** [comment]
would this essentially work on higher versions like 1.20?

**Comment by newxy77** [comment]
@whyimsodiao have you solved this problem now?this also happen to my MCEternal ,and pcl as same

**Comment by whyimsodiao** [comment]
@newxy77  I can not find a way to solve it, and gave my save up. I'm so sorry.

**Comment by newxy77** [comment]
oh that was sad,thanks for your reply,guess I have no way

**Comment by kyraiskool** [comment]
is there anyone here that can explain using this like I'm 5? I've downloaded picocli and nbt. Tho I have no idea how to use the patch file included and everytime I open terminal to run the file I get an error that the file isn't find. I'm really lost T_T

I'm running 1.19.2 and my save of 2 years just corrupted on me... all I did was change some of my configs it's been working just fine until I did that. I'd take it all back if I could
